libs/cli/langchain_cli/integration_template/integration_template/llms.py

```diff
@@ -41,7 +42,7 @@ def _generate(
     ) -> LLMResult:
         raise NotImplementedError
 
-    # TODO: Implement if __model_name__ supports async generation. Otherwise
```

Comment:
unrelated but should we also add a TODO for _generate, since that's the only required TODO

Reply:
added